test/Manager-test.js

```diff
@@ -0,0 +1,54 @@
+import { expect } from 'chai';
+import sampleBookingData from './sample-booking-data'
+import sampleRoomData from './sample-room-data'
+import sampleUserData from './sample-user-data'
+import Manager from '../src/Manager';
+
+describe.only('Manager', () => {
```

Comment:
Nice job setting up sample data! I don't plan to clone your repo to run the tests, but if you want me to, just let me know!!